### PR TITLE
update docs to explain proxy configuration

### PR DESCRIPTION
**Pull request by wwwtyro**
@chuckbutler @mbruzek @Cynerva 

#39 #61 #63

### CODE REVIEW

**Comment by chuckbutler** [comment]
I approve of this message 👍

File: README.md [hunk 1]
```diff
@@ -31,8 +31,34 @@ This bundle is for multi-node deployments, for individual deployments for
...
+appropriately before deploying the Kubernetes bundle.
 
+First, configure your model's `http-proxy` and `https-proxy` settings with your
+proxy (here we use `squid.internal:3128` as an example):
+
+```sh
+$ juju model-config http-proxy=http://squid.internal:3128 https-proxy=https://squid.internal:3128
+```
+
+You'll also need to add `localhost` to the `no-proxy` model configuration
```
Comment (mbruzek) [inline README.md]: Explain what no-proxy does and why they need it.  "You will need to add some hosts that are not proxied, such as localhost, and the address(es) of the local Juju controller."

File: README.md [hunk 2]
```diff
@@ -31,8 +31,34 @@ This bundle is for multi-node deployments, for individual deployments for
 developers, use the smaller
 [kubernetes-core](http://jujucharms.com/kubernetes-core) bundle.
 
-## Deploy the bundle
+## Proxy configuration
+
+If you're operating behind a proxy, you'll need to configure your model
```
Comment (mbruzek) [inline README.md]: How would the users know if they are behind a proxy? Can you write more words here about that? What does being "behind" a proxy mean? You mean the Juju controller or the charms themselves. Be a little more specific.

**Comment by mbruzek** [comment]
Words, you have the best words! +1

**Pull request merged**